test/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@rules_gazebo//gazebo:headers.bzl", "gz_configure_header")
+
+package(
+    default_applicable_licenses = ["//:license"],
+    features = [
+        "layering_check",
+        "parse_headers",
+    ],
+)
+
+gz_configure_header(
```

Comment:
Not needed in this PR, but moving to `@gz-common//testing` should help get rid of this header.